iamidentityv1/iam_identity_v1_examples_test.go

```diff
@@ -981,20 +977,35 @@ var _ = Describe(`IamIdentityV1 Examples Tests`, func() {
 				IfMatch:    &profileEtag,
 			}
 
-			profileIdnetities, response, err := iamIdentityService.SetProfileIdentities(&setProfileIdentitiesOptions)
+			profileIdentities, response, err := iamIdentityService.SetProfileIdentities(&setProfileIdentitiesOptions)
 
 			if err != nil {
 				panic(err)
 			}
-			b, _ := json.MarshalIndent(profileIdnetities, "", "  ")
+			b, _ := json.MarshalIndent(profileIdentities, "", "  ")
 			fmt.Println(string(b))
 
 			// end-set_profile_identities
 			fmt.Printf("\nSetProfileIdentities() response status code: %d\n", response.StatusCode)
 
 			Expect(err).To(BeNil())
 			Expect(response.StatusCode).To(Equal(200))
-			Expect(profileIdnetities).ToNot(BeNil())
+			Expect(profileIdentities).ToNot(BeNil())
+
+			// remove identity so it can be added again in the next test
+			deleteProfileIdentityOptions := iamidentityv1.DeleteProfileIdentityOptions{
+				ProfileID:    &profileId,
+				IdentityType: core.StringPtr("user"),
+				IdentifierID: &iamID,
+			}
+
+			deleteResponse, delErr := iamIdentityService.DeleteProfileIdentity(&deleteProfileIdentityOptions)
+
+			if delErr != nil {
+				panic(delErr)
```

Comment:
Panicking in a `ginkgo` test has no effect - at least not the expected one. The testing framework will recover and continue the execution of the rest. You should simply remove the `if delErr...` code block and let the `Expect(delErr)...` to handle the error scenarios here. :) Unfortunately there is no way to completely stop the execution in this version. However you can utilize `shouldSkipTest` if you want.